docs/Reference/API-Methods.md

```diff
@@ -51,23 +51,27 @@ Adds a [static node](../HowTo/Find-and-Connect/Managing-Peers.md#static-nodes).
     
 ### admin_changeLogLevel
 
-Change the log level without restarting Besu. 
+Changes the log level without restarting Besu. You can change the log level for all logs, or you can change the log level for specific packages or classes.
+
+Only one log level can be specified per RPC call.
 
 **Parameters**
 
 `level` - [Log level](CLI/CLI-Syntax.md#logging)
 
+`log_filter`: `Array` - Packages or classes to change the log level for. Optional.
+
 **Returns**
 
 `result` : `Success` if the log level has changed; otherwise `error`. 
 
 !!! example
     ```bash tab="curl HTTP request"
-    curl -X POST --data '{"jsonrpc":"2.0","method":"admin_changeLogLevel","params":["DEBUG"], "id":1}' http://127.0.0.1:8545
+    curl -X POST --data '{"jsonrpc":"2.0", "method":"admin_changeLogLevel", "params":["DEBUG", ["tech.pegasys.pantheon.ethereum.eth.manager","tech.pegasys.pantheon.ethereum.p2p.rlpx.connections.netty.ApiHandler"]], "id":1}' http://127.0.0.1:8545
```

Comment:
I wonder if it's worth including both examples - one changing the log level for all logs and the new one? Up to you.

Reply:
Yes.  Demo all to WARN and then these to DEBUG would be some heavily copied and pated values.

Reply:
Added a second example as requested.